eth/stagedsync/exec3_serial.go

```diff
@@ -124,6 +124,9 @@ func (se *serialExecutor) execute(ctx context.Context, tasks []*state.TxTask) (c
 			if se.cfg.chainConfig.Bor != nil && txTask.TxIndex >= 1 {
 				// get last receipt and store the last log index + 1
 				lastReceipt := txTask.BlockReceipts[txTask.TxIndex-1]
+				if lastReceipt == nil || lastReceipt.Logs == nil {
+					return false, fmt.Errorf("last receipt log is nil")
```

Comment:
Add block num and/or txnum to error message. Also i’m not sure: is this proper error handling? I mean - will Erigon self-recover from this state? Nil is expected or not?

Reply:
`nil` is not expected and bad. I'm still investigating why this happens.